### PR TITLE
ECCI-439: config alignment

### PR DESCRIPTION
**Pull request by andystone78**

### CODE REVIEW

File: config/default/config_ignore.settings.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 _core:
   default_config_hash: UVH1aJ4b44UM-VdPVN7hNNuuVqfReJxwfVeDQH1Hvsk
 ignored_config_entities:
-  - devel.toolbar.settings
-  - devel.settings
   - fathom_analytics.settings
-  - system.menu.devel
   - system.site
```
Comment (siliconmeadow) [inline config/default/config_ignore.settings.yml:5]: Could be `system.site:page.front` perhaps?

File: config/default/system.image.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 _core:
   default_config_hash: durWHaKeBaq4d9Wpi4RqwADj1OufDepcnJuhVLmKN24
-toolkit: gd
+toolkit: imagemagick
```
Comment (siliconmeadow) [inline config/default/system.image.yml:3]: Out of curiosity, is imagemagick in use on .gov?

File: config/default/config_ignore.settings.yml (hunk 1 above)
Comment (Polynya) [reply config/default/config_ignore.settings.yml]: Changed to system.site:page to catch other pages

Comment (Polynya) [reply config/default/config_ignore.settings.yml]: Changed to system.site:page to catch other pages

File: config/default/system.image.yml (hunk 2 above)
Comment (Polynya) [reply config/default/system.image.yml:3]: It's installed in Dockerfile so it's available

File: config/default/config_ignore.settings.yml (hunk 1 above)
Comment (siliconmeadow) [reply config/default/config_ignore.settings.yml]: Good thinking!

**Review by siliconmeadow — APPROVED** [review]
LGTM

**Pull request merged**